src/main/java/frc/robot/config/FluidConstant.java

```diff
@@ -76,14 +76,6 @@ public T getValue() {
         return this.value;
     }
     
-    /**
-     * Get method for implementing the Supplier interface
-     * @return The current value of this constant
-     */
-    public T get() {
-        return this.value;
-    }
-    

```

Comment:
Was this block deleted intentionally?

Reply:
This block seems to be deleted in many of the open PRs so I suspect it may be occurring automatically by the IDE as it is possibly unused?

Reply:
When I opened the file there were 2 of that method and VSCode was giving an error.